### ISSUE TITLE
False positive on unused arguments when argument used in query param

### ISSUE THREAD

**Issue by justinmclean** [cflint/CFLint#418]
Example code:
```
<cffunction name="lookup" access="public" output="false" returntype="void">
<cfargument name="domain" type="string" required="true" />
<cfset var details = "">
<cfquery name="details" datasource="#request.datasource#">
select *
from domains
where domain = <cfqueryparam value="#domain#" cfsqltype="CF_SQL_VARCHAR">
</cfquery>
</cffunction>
```

**Comment by TheRealAgentK** [comment]
what's the exactly log/error it creates?

**Comment by justinmclean** [comment]
UNUSED_METHOD_ARGUMENT

**Comment by ryaneberly** [comment]
fixed in #416
